project.clj

```diff
@@ -207,6 +208,19 @@
    :run
    [:exclude-tests {}]
 
+   :run-with-repl
+   [:exclude-tests
+    :include-all-drivers
+
+    {:env
+     {:mb-jetty-join "false"}
+
+     :repl-options
+     {:init (do
+             (use 'metabase.core)
```

Comment:
You want `(require 'metabase.core)` here, not `use`. `use` is legacy/deprecated and imports the symbols in that namespace as well (not what you want to do)